collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.java

```diff
@@ -80,10 +83,7 @@ private void initializeFrameworks() {
     }
 
     private void initializeAnalytics() {
-        analytics.setAnalyticsCollectionEnabled(false);
-
-//        FormUpdateMode formUpdateMode = getFormUpdateMode(context, generalSettings);
```

Comment:
Leave this in uncommented below instead?

Reply:
I'm worried it'll produce confusing results, so I added a note in to the analytics issue (#4605) to fix it properly.

Reply:
Got it, great.